src/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -1260,7 +1260,7 @@ private static void AssertHeadersAreInjected(List<string> requestLines, Activity
             {
                 Assert.Null(requestId);
                 Assert.True(traceparent != null, "traceparent was not injected when W3C instrumentation was enabled");
-                Assert.True(traceparent.StartsWith($"00-{parent.TraceId.ToHexString()}-"));
```

Comment:
Why not starts with?

Reply:
Grr.  Should have been StartsWith.  Must have been one of the few I did manually before I started using the auto-fixer.  Will fix.